py/server/deephaven/jcompat.py

```diff
@@ -304,3 +304,24 @@ def _j_array_to_series(dtype: DType, j_array: jpy.JType, conv_null: bool) -> pd.
         s = pd.Series(data=np_array, copy=False)
 
     return s
+
+
+class SafeCloseable:
+    """A context manager wrapper of Java SafeCloseable to emulate Java try-with-resources."""
```

Comment:
Let's follow the JObjectWrapper pattern here.
```suggestion
class SafeCloseable(JObjectWrapper):
    """A context manager wrapper of Java SafeCloseable to emulate Java try-with-resources."""
```